app/views/steps/_form_assets.html.erb

```diff
@@ -1,16 +1,25 @@
-<div class="panel panel-default panel-step-attachment">
-  <div class="panel-heading">
-    <span class="fas fa-paperclip"></span>
-    <%= t("protocols.steps.new.asset_panel_title") %>
-    <div class="pull-right">
-      <% unless ff.object.file.exists? && ff.object.locked? %>
-        <%= ff.remove_nested_fields_link do %>
-          <span class="fas fa-times"></span>
-        <% end %>
-      <% end %>
-    </div>
+<% asset = ff.object %>
+<% image_preview = asset.is_image? ? asset.url(:medium) : nil %>
+
+<div class="attachment-placeholder pull-left">
```

Comment:
This `div` could be dryed up, since it repeats below as well (on `_item.rb`).

Reply:
Actully we can't, because we are rendering _form_asset separete for 'edit' form not on show with `_item.rb` partial.